docs/cck/components/Combat System.md

```diff
@@ -0,0 +1,59 @@
+# Combat System
+Adding this component enables PvP and PVE style gamemodes in your world. Best used alongside the **Damage** Component and the **Game Instance Controller** Component
+
+### Properties
+
+##### Reference ID
+
+##### Health
++ **Health Base Amount:** This is the default Health each player starts with at the begining of a Game or Round.
++ **Health Max Amount:** This is the Maximum amount of Health a player can have at any given time.
+
+##### Health Regeneration
++ **Health Regeneration Delay:** The time in seconds before the player starts regenerating Health.
++ **Health Regeneration Rate:** How much Health the player receives per second.
++ **Health Regeneration Cap:** The amount of Health that regeneration will stop.
+
+##### Armor
++ **Armor Base Amount:** This is the default Armor each player starts with at the begining of a Game or Round.
++ **Armor Max Amount:** This is the Maximum amount of Armor a player can have at any given time.
+
+##### Armor Regeneration
++ **Armor Regeneration Delay:** The time in seconds before the player starts regenerating Armor.
++ **Armor Regeneration Rate:** How much Armor the player receives per second.
++ **Armor Regeneration Cap:** The amount of Armor that regeneration will stop.
```

Comment:
This is worded strangely. Would "The maximum amount of armor that is regenerated." work better here?

Reply:
The armor cap is based on the amount the player has, if uncapped it'll keep giving armor to the player until the maximum armor is reached, similar to health and shield.